### PR TITLE
Zeus

### PR DESCRIPTION
**Pull request by mablue**
Thank you for sending your pull request.

## Summary
Explain in one sentence the goal of this PR / Strategy

Solve the issue: #___

## Quick strategy idea

- <change log #1>
- <change log #2>

### CODE REVIEW

**Comment by snussik** [comment]
@mablue Ive tested a couple of strategies & I think that using `--spaces all` or `--spaces trailing` during the first hyperopt as it's commented in the head is not optimal. I @xmatthias pointed out in [#4925](https://github.com/freqtrade/freqtrade/issues/4925) issue it trailing opt works inside candle which gives some sort of look forward effect on optimisation & backtesting. And I think that I you use it alongside with other hypeopt parameters it tries to always make `buy=True` & `sell=False` as it gives better trailing optimisation.
IMHO, trailing should be the last and separate hypeopt after other parameters were found.

**Comment by mablue** [comment]
> @mablue Ive tested a couple of strategies & I think that using `--spaces all` or `--spaces trailing` during the first hyperopt as it's commented in the head is not optimal. I @xmatthias pointed out in [#4925](https://github.com/freqtrade/freqtrade/issues/4925) issue it trailing opt works inside candle which gives some sort of look forward effect on optimisation & backtesting. And I think that I you use it alongside with other hypeopt parameters it tries to always make `buy=True` & `sell=False` as it gives better trailing optimisation.
> IMHO, trailing should be the last and separate hypeopt after other parameters were found.
 its cuz of that we can determine with our bots that may price will increase. But generating true sell signals need to have some info about btc pump and dumps and have not a technical way too analyzed with just seeing indoor indicators.
If you want to have not problem anymore use --spaces buy trailing roi and after that rehype with just stoploss space and get the result that have not very small stoplosses. For example -0.02 stoploss will make you a big shit bro.
@xmatthias will think about that I mentioned befour you that problem.
 if we have an stoploss ignorance time gap parameter parameter we can handle small stoplosses.
For example:
Stoploss_ignorance_time='1h'

**Review by xmatthias — APPROVED** [review]

**Comment by rajmondx** [comment]
[`strategies/lookahead_bias/readme.md`](https://github.com/freqtrade/freqtrade-strategies/blob/main/user_data/strategies/lookahead_bias/readme.md) says:

----------------------------------------------------------
Warning, Strategies in this folder do have a lookahead bias.

Please see these as practice to see if you can spot the lookahead bias.

<details>
<summary>DevilStra</summary>

`normalize()` uses `.min()` and `.max()`. This uses the full dataframe, not just past data.

</details>

<details>
<summary>GodStraNew</summary>

`normalize()` uses `.min()` and `.max()`. This uses the full dataframe, not just past data.
</details>
<details>
<summary>Zeus</summary>

uses `.min()` and `.max()` to normalize `trend_ichimoku_base` as well as `trend_kst_diff`.

</details>

<details>
<summary>wtc</summary>

``` python
min_max_scaler = preprocessing.MinMaxScaler()
x_scaled = min_max_scaler.fit_transform(x)
```

Using a MinMaxScaler will automatically take the absolute maximum and minimum of a series.

</details>
</details>
----------------------------------------------------------

Consider those statements when you review backtesting results.

Btw. the Zeus strategy is currently broken anyway (didn't take the time to check why).
